adapter/internal/oasparser/envoyconf/envoyconf_internal_test.go

```diff
@@ -86,11 +87,6 @@ func TestCreateRoute(t *testing.T) {
 	}
 	regexRewriteWithXWso2BasePath := &envoy_type_matcherv3.RegexMatchAndSubstitute{
 		Pattern: &envoy_type_matcherv3.RegexMatcher{
-			EngineType: &envoy_type_matcherv3.RegexMatcher_GoogleRe2{
```

Comment:
shouldn't we keep the EngineType as it is?

Reply:
https://www.envoyproxy.io/docs/envoy/v1.23.3/api-v3/config/bootstrap/v3/bootstrap.proto#envoy-v3-api-field-config-bootstrap-v3-bootstrap-default-regex-engine